deps/build.jl

```diff
@@ -0,0 +1,36 @@
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
```

Comment:
Shouldn't this be expressed by version bounds in REQUIRE, rather than as a comment?